src/GuiBase/Timeline/Timeline.h

```diff
@@ -0,0 +1,330 @@
+/*!
+ * \brief Custom Qt widget for animation (timeline)
+ * \author Gauthier Bouyjou (email : gauthierbouyjou@aol.com)
+ * \date april 2019
+ *
+ * For question send me mail or add issue to initial
+ * github repo : https://github.com/hiergaut/AnimTimeline.git
+ * current commit id : aaa363ee8ec0247e9299a8949e5bd334b10dc413
+ */
+
+// TODO / FIXME : CHECK LICENCE
```

Comment:
Is it still needed. Interns that work on Radium must sign a copyleft document. So The licens is the same as the Radium one.